modman

```diff
@@ -1,2 +0,0 @@
-app/code/community/Meanbee/Footerjs/ app/code/community/Meanbee/Footerjs/
```

Comment:
What problem did you run into using modman in collaboration with composer? I'm not comfortable moving from support one installation method to another. Composer isn't standard for many on M1.